lms/djangoapps/courseware/user_state_client.py

```diff
@@ -166,12 +114,6 @@ def _get_student_modules(self, username, block_keys):
                 usage_key = student_module.module_state_key.map_into_course(student_module.course_id)
                 yield (student_module, usage_key)
 
-    @contract(
-        username="basestring",
-        block_keys="seq(UsageKey)|set(UsageKey)",
-        scope=ScopeBase,
-        fields="seq(basestring)|set(basestring)|None"
-    )
     def get_many(self, username, block_keys, scope=Scope.user_state, fields=None):
         """
         Retrieve the stored XBlock state for a single xblock usage.
```

Comment:
Not part of the code/doc that has been changed but the doc here should be updated. "Retrieve the store XBlock states for xblock uses" ?